### PR TITLE
feat(event): description section

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

File: packages/react-components/src/molecules/__tests__/EventDescription.test.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React, { ComponentProps } from 'react';
+import { render } from '@testing-library/react';
+import { createEventResponse } from '@asap-hub/fixtures';
+
+import EventDescription from '../EventDescription';
+
+jest.mock('../../localization');
```
Comment (jeysal) [inline packages/react-components/src/molecules/__tests__/EventDescription.test.tsx:7]: y dis

File: packages/react-components/src/molecules/EventDescription.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { EventResponse } from '@asap-hub/model';
+import TagList from './TagList';
+import { RichText } from '../organisms';
+import { Headline4, Divider } from '../atoms';
+
+type EventInfoProps = Pick<EventResponse, 'tags' | 'description'>;
+
+const EventDescription: React.FC<EventInfoProps> = ({ tags, description }) => {
```
Comment (jeysal) [inline packages/react-components/src/molecules/EventDescription.tsx:9]: headlines should be level 2 styled as 4

**Review by jeysal — CHANGES_REQUESTED** [review]
kool

File: packages/react-components/src/molecules/__tests__/EventDescription.test.tsx (hunk 1 above)
Comment (kmaid) [reply packages/react-components/src/molecules/__tests__/EventDescription.test.tsx:7]: I wondered this too and then didn't remove it xD

**Review by jeysal — APPROVED** [review]

**Comment by jeysal** [comment]
rebasing

**Pull request merged**